store/sqlstore/supplier.go

```diff
@@ -114,6 +113,16 @@ type SqlSupplier struct {
 	lockedToMaster bool
 }
 
+type TraceOnAdapter struct{}
+
+func (t *TraceOnAdapter) Printf(format string, v ...interface{}) {
+	originalString := fmt.Sprintf(format, v...)
+	newString := strings.ReplaceAll(originalString, "\n", " ")
+	newString = strings.ReplaceAll(newString, "\t", "")
```

Comment:
I think there's a potential to convert valid SQL to invalid SQL. See https://play.golang.org/p/c-2-8xPeqgP as a demonstration.
```suggestion
	newString = strings.ReplaceAll(newString, "\t", " ")
```